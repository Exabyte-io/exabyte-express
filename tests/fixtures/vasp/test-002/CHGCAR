version https://git-lfs.github.com/spec/v1
oid sha256:bb1230a0c4d0fdce7bc43bbcbe86cbcb233622ec5ceee73c2c1cd157361aef2f
size 2023370
