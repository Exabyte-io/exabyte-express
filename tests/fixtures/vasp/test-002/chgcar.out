version https://git-lfs.github.com/spec/v1
oid sha256:146d7cf7541d79d6baec45aec4541c2a8d687dc5b301c08b77b4a7ad39f15ede
size 1597
